lib/crypto.js

```diff
@@ -18,6 +18,7 @@ let encryption, signing;
  * @see https://tools.ietf.org/html/rfc5869
  *
  */
+/* istanbul ignore else */
```

Comment:
This doesn't get hit since I upgraded the circleci Node image - will go back and setup the CI to run on multiple Node versions in another PR